RoslynPluginGenerator/AnalyzerPluginGenerator.cs

```diff
@@ -314,7 +313,7 @@ private bool IsValidSqaleFile(string sqaleFilePath)
             return true;
         }
 
-        private static PluginManifest CreatePluginManifest(IPackage package)
+        public /* for test */ static PluginManifest CreatePluginManifest(IPackage package)
```

Comment:
It seemed to be neatest and easiest to open CreatePluginManifest for direct testing.